### PR TITLE
Jim Dechant 1710 BEE

### PR DESCRIPTION
**Pull request by Ecksi**
End of Iteration 2

### CODE REVIEW

File: lib/activity.rb [hunk 1]
```diff
@@ -1,2 +1,32 @@
 class Activity
+  attr_reader :activity,
+              :participants
+
+  def initialize(activity, participants)
+    @activity      = {activity => participants}
+    @participants  = participants
+    @amounts_owed  = []
+  end
```
Comment (VictoriaVasys) [inline lib/activity.rb]: The spec specifically stated that an activity has a `name`. I'm also a little confused on your `@activity` variable; why use that if you have the participants hash available?

File: lib/activity.rb [hunk 2]
```diff
@@ -1,2 +1,32 @@
...
+
+  def add_participants(activity, participant)
+    @participants[:name] << participant
+  end
+
+  def cost_of_activity(activity)
+    @activity["#{activity}"][:amount_paid]
```
Comment (VictoriaVasys) [inline lib/activity.rb]: Using interpolation here is redundant; `activity` is passed in as a string; if you're trying to incorporate an edge case, it's better to use `to_s`.

File: lib/activity.rb [hunk 3]
```diff
@@ -1,2 +1,32 @@
...
+  end
+
+  def split_cost(activity)
+    foo = @activity["#{activity}"][:name].count
+    bar = @activity["#{activity}"][:amount_paid]
+
+    bar / foo
+  end
```
Comment (VictoriaVasys) [inline lib/activity.rb]: Be sure to use Ruby convention when naming variables. What does each of those variables actually represent?

File: lib/activity.rb [hunk 4]
```diff
@@ -1,2 +1,32 @@
...
+
+  def add_to_amount_owed(activity)
+    baz = @activity["#{activity}"][:name]
+    baz.each do |index|
+      puts "#{index} owes #{split_cost(activity)}"
+    end
+  end
```
Comment (VictoriaVasys) [inline lib/activity.rb]: The name of this method indicates that you're adding to some collection called amount_owed, when you're actually printing. Also, we shouldn't test methods that return `puts` statements since they return nil. I'd advise against printing to the console unless the spec specifically asks for it.
